examples/pipelines/fl_cross_silo_factory/submit.py

```diff
@@ -190,13 +180,33 @@ def connect_to_aml():
 
 
 #########################
-### C. CUSTOM DS CODE ###
+### C. Scatter/Gather pipelines ###
 #########################
 
 # This is your section, please modify anything here following the guidelines
 # in the docstrings.
 
 
+@pipeline
+def gather_pipeline(
```

Comment:
should we just use aggregate_component directly?

Reply:
done